pkg/config/appsec.go

```diff
@@ -5,9 +5,13 @@
 
 package config
 
+import traceconfig "github.com/DataDog/datadog-agent/pkg/trace/config"
+
 // setupAppSec initializes the configuration values of the appsec agent.
 func setupAppSec(cfg Config) {
-	cfg.BindEnvAndSetDefault("appsec_config.enabled", true, "DD_APPSEC_ENABLED")
-	cfg.BindEnvAndSetDefault("appsec_config.appsec_dd_url", "", "DD_APPSEC_DD_URL")
-	cfg.BindEnvAndSetDefault("appsec_config.max_payload_size", 5*1024*1024, "DD_APPSEC_MAX_PAYLOAD_SIZE")
+	cfg.BindEnvAndSetDefault("appsec_config.enabled", traceconfig.DefaultAppSecEnabled, "DD_APPSEC_ENABLED")
```

Comment:
I still don't like these constants. Sorry. I don't want to sound like a beaten drum. We have the same situation in other places to, and we don't use constants there. I wouldn't mind if we added constants for everything, but having them just for AppSec is weird. Especially now when it's a module, it's a weird API to have these constants there.

I'd like to suggest two more ways to avoid them:
* Set them in line (hard coded) into `traceconfig.New()`. Run `defaults := traceconfig.New()` here and pull them.
* Only use `BindEnv` and not `BindEnvAndSetDefault`. Then, the defaults from `New()` will be kept.

Reply:
Done in https://github.com/DataDog/datadog-agent/pull/11324/commits/1e8cf3f86bfb9e1eb7a9c4b5cd3e8e7fd102187a